src/components/document/FamilyMeta.tsx

```diff
@@ -22,7 +22,7 @@ export const FamilyMeta = ({ category, date, geographies, topics, author, corpus
   return (
     <>
       <CountriesLink geographies={geographies} countries={countries} />
-      {!isNaN(year) && <span data-cy="family-metadata-year">Approval FY: {year}</span>}
+      {!isNaN(year) && <span data-cy="family-metadata-year">{`${category === "MCF" ? "Approval FY: " + year : year}`}</span>}
```

Comment:
Can you add a TODO comment that we will need to circle back and look at this later once we get the config more established. Because we won't be able to have conditional checks like this in the near future.